lib/experimental/keymap.dart

```diff
@@ -0,0 +1,26 @@
+// HTML for keyboard shortcuts dialog
+import 'dart:html';
+
+import 'package:dart_pad/core/keys.dart';
+
+String keyMapToHtml(Map<Action, Set<String>> keyMap) {
+  DListElement dl = DListElement();
+  keyMap.forEach((Action action, Set<String> keys) {
+    if (!action.hidden) {
+      String string = '';
```

Comment:
Unlikely to make a noticeable performance difference, but this seems like a good place to use a StringBuffer.

Reply:
this code is just moving from lib/new_playground.dart, I'm not sure a StringBuffer is necessary